service/controller/key/key.go

```diff
@@ -120,7 +120,7 @@ func CommonAppSpecs() []AppSpec {
 			Chart:           "kube-state-metrics-app",
 			Namespace:       metav1.NamespaceSystem,
 			UseUpgradeForce: true,
-			Version:         "0.6.0",
+			Version:         "1.0.0",
```

Comment:
This didn't get updated before but it ensures all default apps are on 1.X. So I've included it.